data/human/human missions.txt

```diff
@@ -107,7 +107,269 @@ mission "Pookie, Part 2"
+			`	You feel your back get nudged, and turn to see a woman dressed in rags holding four buckets filled with credits. The names of the racers are crudely written on each of them. "Betting is 3,000 credits," she says unusually sweetly.`
+			branch notpoor
+				"credits" >= 3000
+			choice
+				`	(Bet on Bleedbug.)`
+				`	(Bet on Doom.)`
+				`	(Bet on Zinc.)`
+				`	(Bet on Puny.)`
+				`	"I'll pass."`
+					goto nobet
+			`	You dig through your pockets for a credit chip to toss into the bucket, until you realize that you don't have enough money to make a bet. Impatient with your bumbling, the woman turns to take money from a person next to you.`
+				goto next
+			label notpoor
+			choice
+				`	(Bet on Bleedbug.)`
+				`	(Bet on Doom.)`
+				`	(Bet on Zinc.)`
+				`	(Bet on Puny.)`
+				`	"I'll pass."`
+					goto nobet
+			action
+				set "bet on hound racing"
+				payment -3000
+			`	You toss a chip into the bucket, and the woman eagerly turns towards a person next to you.`
+				goto next
+			label nobet
+			`	The woman turns to take bets from a person next to you.`
+			label next
+			`	After several minutes pass, the siren sounds again, and the announcer speaks again. "Once again, we at the East Mesa Racing Complex give our thanks to everyone who supports our cause by participating in betting. Now, without further ado..."`
+			`	A red light on the ceiling turns on, and the audience shouts, "Three!" A second one lights up. "Two!" A third light, this time yellow, turns on. "One!"`
+			`	A green light appears, and the crowd descends into chaos as the cages' doors open.`
+			`	The first to move is Bleedbug, desperate to get as far away from Doom as possible. Doom himself has his eyes set on a different target. As soon as the doors open, he rushes over the lane lines towards Zinc and bares his teeth for a bite. Anticipating this, Zinc dodges beneath the attack, and lunges in with her mouth open. Doom responds to the threat by lowering his head and bashing into her, and the two animals are quickly locked into a brutal mess of teeth and blood. The crowd goes wild at the spectacle unfolding before them as the brawl approaches your section of the fence.`
+			choice
+				`	(Continue watching.)`
+				`	(Get away.)`
+					goto leave
+			`	Looking on, you are suddenly thrown backward when the beasts smash into the fence in front of you. Lying on the ground, you look down and see Zinc and Doom tearing into each other just beyond your feet.`
+			choice
+				`	(Get up and run.)`
```

Comment:
Running sound a bit overreacting (in a room full of people), just moving back to a safer distance should be sufficient to get out of harms way.

Reply:
With a massive wolf and dog in front of you, running doesn't actually sound like too big an overreaction to me.

Reply:
> With a massive wolf and dog in front of you, running doesn't actually sound like too big an overreaction to me.

I agree, but the place was introduced like this: **"The inside of the racing complex is packed to capacity. Around you is a swarm of elbows and shoulders, jutting into you while worming your way through the crowd."**   This doesn't sound like a place where there is space to run. I do agree that "quickly increasing the distance between you and the massive beasts" is a proper reaction; it just cannot be called running when first-last is moving through a massive crowd of people.